lib/web_ui/dev/felt

```diff
@@ -133,28 +129,5 @@ then
   set -e
 fi
 
-if [[ "$FELT_USE_SNAPSHOT" == "false" || "$FELT_USE_SNAPSHOT" == "0" ]]; then
-  echo "[Snapshot mode: off]"
-  # Running without snapshot means there is high likelihood of local changes. In
-  # that case, let's clear the snapshot to avoid any surprises.
-  rm -f "$SNAPSHOT_PATH"
-  rm -f "$STAMP_PATH"
-  install_deps
-  $DART_SDK_DIR/bin/dart $FELT_DEBUG_FLAGS "$DEV_DIR/felt.dart" $@
-else
-  # Create a new snapshot if any of the following is true:
-  #  * SNAPSHOT_PATH is not a file, or
-  #  * STAMP_PATH is not a file with nonzero size, or
-  #  * Contents of STAMP_PATH is not our local git HEAD revision, or
-  #  * pubspec.yaml last modified after pubspec.lock
-  if [[ ! -f $SNAPSHOT_PATH || ! -s "$STAMP_PATH" || "$(cat "$STAMP_PATH")" != "$REVISION" || "$WEB_UI_DIR/pubspec.yaml" -nt "$WEB_UI_DIR/pubspec.lock" ]]; then
-    echo "[Snapshot mode: on] (creating a new snapshot)"
-    install_deps
-    mkdir -p $DART_TOOL_DIR
-
-    "$DART_SDK_DIR/bin/dart" --snapshot="$SNAPSHOT_PATH" --packages="$WEB_UI_DIR/.dart_tool/package_config.json" "$SCRIPT_PATH"
-    echo "$REVISION" > "$STAMP_PATH"
-  fi
-
-  $DART_SDK_DIR/bin/dart $FELT_DEBUG_FLAGS --packages="$WEB_UI_DIR/.dart_tool/package_config.json" "$SNAPSHOT_PATH" $@
-fi
+install_deps
+$DART_SDK_DIR/bin/dart $FELT_DEBUG_FLAGS --packages="$WEB_UI_DIR/.dart_tool/package_config.json" "$SCRIPT_PATH" $@
```

Comment:
If we use `dart run` could we avoid `install_deps` and needing to pass the `--packages` directly? https://dart.dev/tools/dart-run

Reply:
It looks like even with `dart run` I have to run `install_deps` first (it doesn't seem like `dart run` pulls dependencies. But I can change over to `dart run` to remove the need for the --packages stuff.